static/app/views/performance/transactionSummary/transactionOverview/content.tsx

```diff
@@ -230,7 +227,7 @@ function SummaryContent({
         group.vitals.some(vital => {
           const functionName = `percentile(${vital},${VITAL_PERCENTILE})`;
           const field = functionName;
-          return Number.isFinite(totalValues[field]);
+          return Number.isFinite(totalValues[field]) && totalValues[field] !== 0;
```

Comment:
this check would fail when `<web_vital>.measurements` would get returned as `0` (metrics) instead of `null` (indexed)